subliminal/cli.py

```diff
@@ -181,7 +181,7 @@ def plural(quantity: int, name: str, *, bold: bool = True, **kwargs: Any) -> str
 )
 @click.option(
     '-c',
-    '--config',
+    '--config-file',
```

Comment:
any reason for the rename? imho `--config` fits better

Reply:
I think it's more accurate because it expects a file path. Why do you prefer `--config` ?

Reply:
it feels somewhat more consistent with most other software that i've used, but i guess it's just a personal preference

Reply:
At the time, I checked other softwares and I had the feeling that `--config-file` was more used :laughing:
Using `--config` makes it more general, so we don't have to change the name if at some point we want to allow `KEY=VALUE` for instance. `ruff` is like that.
`mypy` uses `--config-file`.

Reply:
I'll revert then :+1: